### PR TITLE
[AIFBR-12934] Added script for updating content uri after aic external storage migration

### PR DESCRIPTION
**Pull request by aakashuipath**

### CODE REVIEW

File: platform/externalstoragemigration/content_uri_update_script.sh [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            echo "Query successfully executed"
+    fi
+
+}
+
+STORAGETYPE=$1
+HOST=$2
+USERNAME=$3
+PASSOWORD=$4
+DBNAME=$5
```
Comment (narendrakumar-nj) [inline platform/externalstoragemigration/content_uri_update_script.sh:95]: This block is duplicated at the top

File: platform/externalstoragemigration/content_uri_update_script.sh [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+
+
+check_input_value() {
```
Comment (SahilChaudhary25) [inline platform/externalstoragemigration/content_uri_update_script.sh:16]: nit add function keyword here

File: platform/externalstoragemigration/content_uri_update_script.sh [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+yellow=$(tput setaf 3)
+default=$(tput sgr0)
+
+STORAGETYPE=$1
+HOST=$2
+USERNAME=$3
+PASSOWORD=$4
+DBNAME=$5
+
+
```
Comment (SahilChaudhary25) [inline platform/externalstoragemigration/content_uri_update_script.sh:14]: nit: remove extra space

File: platform/externalstoragemigration/content_uri_update_script.sh [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    exit 1;
+    fi
+  else
+    echo "$green $(date) $1 is provided $default"
+  fi
+}
+
+
+# function to validate database connection
+validate_dbconn() {
```
Comment (SahilChaudhary25) [inline platform/externalstoragemigration/content_uri_update_script.sh:35]: nit function keyword here

**Review by SahilChaudhary25 — CHANGES_REQUESTED** [review]
Add validate check if all required tool installed before proceeding for query execution

File: platform/externalstoragemigration/content_uri_update_script.sh [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+   
+    echo "$green Validating connection to $HOST $default"
+
+    result=$(sqlcmd -S ${HOST} -U ${USERNAME} -P ${PASSOWORD} -d ${DBNAME} -Q "SELECT DB_NAME()" )
```
Comment (vaibhavs-uipath) [inline platform/externalstoragemigration/content_uri_update_script.sh:39]: check if sqlcmd and other dependency are already installed on the system

File: platform/externalstoragemigration/content_uri_update_script.sh [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+function update_content_uri_azure() {
+
+echo "$green Executing updated content uri in azure in $HOST $default"
+
+STORAGE_HOST=$(kubectl -n uipath get secret aicenter-external-storage-secret -o json | jq '.data.ACCOUNTNAME' | sed -e 's/^"//' -e 's/"$//' | base64 -d)
+
+STORAGE_HOST_SUFFIX=$(kubectl -n uipath get secret aicenter-external-storage-secret -o json | jq '.data.AZURE_FQDN_SUFFIX' | sed -e 's/^"//' -e 's/"$//' | base64 -d)
+
+BUCKET_NAME=$(kubectl -n uipath get secret aicenter-external-storage-secret -o json | jq '.data.BUCKET' | sed -e 's/^"//' -e 's/"$//' | base64 -d)
```
Comment (vaibhavs-uipath) [inline platform/externalstoragemigration/content_uri_update_script.sh:59]: Add null/empty check for these three STROAGE_HOST AND BUCKET_NAME

**Comment by vaibhavs-uipath** [comment]
@aakashuipath Add readme file.

File: platform/externalstoragemigration/content_uri_update_script.sh (hunk 1 above)
Comment (aakashuipath) [reply platform/externalstoragemigration/content_uri_update_script.sh:95]: done

File: platform/externalstoragemigration/content_uri_update_script.sh (hunk 2 above)
Comment (aakashuipath) [reply platform/externalstoragemigration/content_uri_update_script.sh:16]: done

File: platform/externalstoragemigration/content_uri_update_script.sh (hunk 3 above)
Comment (aakashuipath) [reply platform/externalstoragemigration/content_uri_update_script.sh:14]: done

File: platform/externalstoragemigration/content_uri_update_script.sh (hunk 4 above)
Comment (aakashuipath) [reply platform/externalstoragemigration/content_uri_update_script.sh:35]: done

File: platform/externalstoragemigration/content_uri_update_script.sh (hunk 5 above)
Comment (aakashuipath) [reply platform/externalstoragemigration/content_uri_update_script.sh:39]: done

File: platform/externalstoragemigration/content_uri_update_script.sh (hunk 6 above)
Comment (aakashuipath) [reply platform/externalstoragemigration/content_uri_update_script.sh:59]: done

**Review by narendrakumar-nj — DISMISSED** [review]

**Review by narendrakumar-nj — APPROVED** [review]

**Review by vaibhavs-uipath — APPROVED** [review]

**Pull request merged**